bccsp/schemes/dlog/crypto/signature.go

```diff
@@ -430,9 +533,16 @@ func finalise(
 	// disclosed attributes
 	// message being signed
 	// the amount of bytes needed for the nonrevocation proof
-	pdl := len([]byte(signLabel)) + 7*(2*curve.FieldBytes+1) + curve.FieldBytes + len(Disclosure) + len(msg) + ProofBytes[RevocationAlgorithm(cri.RevocationAlg)]
-	if sigType == opts.EidNym {
-		pdl = len([]byte(signWithEidNymLabel)) + 9*(2*curve.FieldBytes+1) + curve.FieldBytes + len(Disclosure) + len(msg) + ProofBytes[RevocationAlgorithm(cri.RevocationAlg)]
+	pdl := curve.FieldBytes + len(Disclosure) + len(msg) + ProofBytes[RevocationAlgorithm(cri.RevocationAlg)]
```

Comment:
I was thinking more about a meticulous description of the payload we're hashing: what is the content in which order

Reply:
Not sure I understand what type of description you're suggesting?

Reply:
I was thinking that it would be helpful to list all the different elements that get hashed, in order. For instance, something like

```
The content of the hashed payload is:
1. the commitment `t1`
2. the commitment `t2`
3. ....
```